charts/sonarqube-lts/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [2.0.0]
+* This chart is no longer maintained. Please use the 'sonarqube' chart to install the new SonarQube 9.9 LTS
```

Comment:
Not sure we should add information for the users in a changelog. I find this weird and sounds to me against the goal of a changelog. I suggest that we remove the "Please..." part.

Reply:
Maybe be simply: the sonarqube chart should be used instead ?

Reply:
Yup, let's change like this :) @jCOTINEAU I propose to rephrase the second sentence in something like "The `sonarqube` chart should be used to install the new SonarQube 9.9 LTS."? WDYT?